hello-1/test_package/conanfile.py

```diff
@@ -1,14 +1,16 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 import os
 
 class PkgTestConan(ConanFile):
-    requires = "hello/0.1@armin/testing" % (username, channel)
+    requires = "hello/0.1@armin/testing"
     generators = "qmake"
+    settings = "os", "compiler", "arch", "build_type"
 
     def build(self):
-        os.chdir(self.conanfile_directory)
-        self.run('qmake')
-        self.run("make")
+        self.run('qmake ../.. -tp vc')
```

Comment:
I am not sure for the syntax for out-of-soruce qmake builds. I am running qmake in a temp (test_package/build/sha) folder, but the project sources are two dirs above, and I want to reference it. This works, but maybe there is some more qmake, elegant way, independent of the temporary build dir location